wallet/src/de/schildbach/wallet/data/DashPayProfileDao.kt

```diff
@@ -17,6 +17,12 @@ interface DashPayProfileDao {
     fun loadDistinct(userId: String):
             LiveData<DashPayProfile?> = load(userId).getDistinct()
 
+    @Query("SELECT * FROM dashpay_profile where username = :username")
+    fun loadFromUsername(username: String): LiveData<DashPayProfile?>
+
+    fun loadFromUsernameDistinct(userId: String):
+            LiveData<DashPayProfile?> = loadFromUsername(userId).getDistinct()
+
```

Comment:
This looks like my code.  It doesn't match that of `DashPayProfileDaoAsync` which has better names.

These methods should be called `loadByUsername` and `loadByUsernameDistinct`

We are missing a `loadByUserId` here compared to `DashPayProfileDaoAsync`, but we also have no use for it currently.

Reply:
Missing `loadByUserId` added